checklists/appsvc_checklist.json

```diff
@@ -0,0 +1,304 @@
+{
+  "$schema": "checklist.schema.json",
+  "items": [
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Outbound network access should be controlled",
+      "description": "Control outbound network access using a combination of regional VNet integration, network security groups and UDR's.  Traffic should be routed to an NVA such as Azure Firewall.  Ensure to monitor the Firewall's logs.",
+      "guid": "c12159e1-14b9-433d-b574-ecccd9bd3baf",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/overview-vnet-integration"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Ensure a stable IP for outbound communications towards internet adresses",
+      "description": "You can provide a stable outbound IP by using VNet integration and using a VNet NAT Gateway or an NVA like Azure Firewall.  This allows the receiving party to allow-list based on IP, should that be needed.  Note that for communications towards Azure Services often there's no need to depend on the IP address and mechanics like Service Endpoints should be used instead.  (Also the use of private endpoints on the receiving end avoids for SNAT to happen and provides a stable outbound IP range.)",
+      "guid": "cda3b54e-b2eb-403d-b9a2-582718d2ddb1",
+      "severity": "Low",
+      "link": "https://docs.microsoft.com/azure/app-service/networking/nat-gateway-integration"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Inbound network access should be controlled",
```

Comment:
Maybe add a similar check for the SCM site

Reply:
Added a note calling this out in the same item.